### ISSUE TITLE
Allow for multiple packages

### ISSUE THREAD

**Issue by jaxxstorm** [pulumi/registry#3066]
Currently, if someone builds a Pulumi provider or package, they have to be careful to namespace the package correctly. If someone else comes along and builds another package with the same name, they are effectively unable to contribute or publish their package.

**Comment by ringods** [comment]
Related: #1217

**Comment by lukehoban** [comment]
Yeah - I think this is a dupe of https://github.com/pulumi/registry/issues/1217. @jaxxstorm anytjing you were hoping to capture here outside of namespacing packages?
